frontend/src/components/ServiceForm/ServiceForm.tsx

```diff
@@ -45,6 +59,28 @@ export const ServiceForm: React.FC<Props> = ({ service, target = DEFAULT_TARGET,
   const appType = findType(applicationTypes, form.type)
   const css = useStyles()
 
+  const IPAndPort = /^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5]):[0-9]+$/
+
+  const checkPort = () => {
+    console.log(IPAndPort.test(`${form.hostname}:${form.port}`))
+    if (IPAndPort.test(`${form.hostname}:${form.port}`)) {
+      backend.set({ loading: true })
+      emit('reachablePort', { port: form.port, host: form?.hostname })
+    } else {
+      backend.set({ reachablePort: false })
+    }
+  }
```

Comment:
I have made a grave mistake in asking for the port:ip validation. Problem is that it now doesn't work if you use 'localhost' or a local mdns address like 'remoteitpi:29999'. So I'm afraid we'll have to go back to just checking on change. Sorry